server/aws-lsp-codewhisperer/src/language-server/localProjectContext/localProjectContextServer.ts

```diff
@@ -4,20 +4,19 @@ import { TelemetryService } from '../../shared/telemetry/telemetryService'
 import { LocalProjectContextController } from '../../shared/localProjectContextController'
 import { languageByExtension } from '../../shared/languageDetection'
 import { AmazonQWorkspaceConfig } from '../../shared/amazonQServiceManager/configurationUtils'
-import { getWorkspaceFolders } from '../../shared/initializeUtils'
 import { URI } from 'vscode-uri'
 
 export const LocalProjectContextServer =
     (): Server =>
-    ({ credentialsProvider, telemetry, logging, lsp }) => {
+    ({ credentialsProvider, telemetry, logging, lsp, workspace }) => {
         let localProjectContextController: LocalProjectContextController
         let amazonQServiceManager: AmazonQTokenServiceManager
         let telemetryService: TelemetryService
 
         let localProjectContextEnabled: boolean = false
 
         lsp.addInitializer((params: InitializeParams) => {
-            const workspaceFolders = getWorkspaceFolders(logging, params)
+            const workspaceFolders = workspace.getAllWorkspaceFolders() || params.workspaceFolders
```

Comment:
Should this be `??` I'm a little rusty...

Similarly in workspaceContextServer.ts

Reply:
From what I could find, it seems like it the right hand side would be used as fallback when left-hand side is any falsy value (including false, 0, "", null, undefined, and NaN). That seemed fine to me